### PR TITLE
Changes to fix #308

### LINKED ISSUE #308: Using output file path might not pickup main ddc log
If using the `--output-file` option on the CLI the user might find if they arent in the base directory of that path, then the `ddc.log` might not be picked up.

For example if I am in `/apps` and run

```
./bin/ddc -n my-namespace --output-file /data/diagnostics/ddc.out.tgz`
```

But If my `$PWD` is ` /data/diagnostics` then this problem wont occur.

This is a kind of edge case and the user is not missing the `ddc.log` at all its just elsewhere.

### PR DESCRIPTION
**Pull request by markcurtis1970**
Also added some more tests, manually tested with:

- set `--output-file`to a diff location from when `CWD` and checked that the archive includes it 
- didnt set `--output-file` and it runs ok

The only thing that can happen is a empty `ddc.log` is created in the `CWD` when the `init` is run during normal usage - I dont see this as a problem though?

### CODE REVIEW

File: pkg/simplelog/logger.go [hunk 1]
```diff
@@ -94,6 +94,22 @@ func InitLoggerWithFile(fileName string) {
 	})
 }
 
+func InitLoggerWithOutputDir(outputDir string) {
```
Comment (rsvihladremio) [inline pkg/simplelog/logger.go:97]: can we remove the old initlogger functions no longer in use?

**Review by rsvihladremio — CHANGES_REQUESTED** [review]

**Review by rsvihladremio — APPROVED** [review]
